src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -13,12 +15,36 @@ import java.time.format.DateTimeFormatter
 class NotifyService(
   @Value("\${notify.enabled}") private val enabled: Boolean,
   @Value("\${notify.templates.licenceApproved}") private val licenceApprovedTemplateId: String,
+  @Value("\${notify.templates.variationForApproval}") private val variationForApprovalTemplateId: String,
   @Value("\${notify.templates.initialLicencePrompt}") private val initialLicencePromptTemplateId: String,
   @Value("\${notify.templates.urgentLicencePrompt}") private val urgentLicencePromptTemplateId: String,
-  private val client: NotificationClient
+  private val client: NotificationClient,
+  private val pduHeadProperties: PduHeadProperties,
 ) {
   fun sendLicenceApprovedEmail(emailAddress: String, values: Map<String, String>, reference: String) {
-    sendEmail(licenceApprovedTemplateId, emailAddress, values, reference)
+    // Hobbled this email - no current requirement to send this out - blanked out the email address to avoid sending
+    // sendEmail(licenceApprovedTemplateId, emailAddress, values, reference)
+    sendEmail(licenceApprovedTemplateId, "", values, reference)
+  }
+
+  fun sendVariationForApprovalEmail(pduCode: String, licenceId: String, firstName: String, lastName: String) {
+    val pduHead: EmailConfig?
+    try {
+      // TODO: Add environment-specific link to approval cases or specific page to approve this variation
+      pduHead = pduHeadProperties.contacts.getValue(pduCode)
+      if (pduHead.email.isNotBlank()) {
+        val values: Map<String, String> = mapOf(
+          Pair("pduHeadFirstName", pduHead.forename),
+          Pair("licenceFirstName", firstName),
+          Pair("licenceLastName", lastName),
+        )
+        sendEmail(variationForApprovalTemplateId, pduHead.email, values, null)
+      } else {
+        log.error("sendVariationForApproval: An email address was not configured for the head of PDU $pduCode")
+      }
+    } catch (e: NoSuchElementException) {
+      log.error("sendVariationForApproval: No PDU head contact detail configured for $pduCode - No email was sent")
```

Comment:
Instead of a try catch, is there not an `orElse()` method you can do?

Reply:
Yes, good spot. There is a `getOrDefault()` which is a bit neater.

Reply:
Done and pushed now.